src/components/schedule/Schedule.js

```diff
@@ -8,26 +8,29 @@ import { getShowInPrismic } from "../../helpers/PrismicHelper";
 const Schedule = ({ scheduleData, residentsData }) => {
   return (
     <Wrapper>
-      <ComingUpText>Coming up...</ComingUpText>
-      <ScheduleItemsWrapper>
-        {scheduleData &&
-          scheduleData.map((scheduleItemData, i) => {
-            const foundShow = getShowInPrismic({
-              residentsData,
-              currentShow: scheduleItemData,
-            });
+      {scheduleData && scheduleData.length ? (
```

Comment:
`scheduleData && !!scheduleData.length &&` might be a away to dodge the ternary?

Reply:
Gonna keep ternary for [these reasons](https://github.com/RichardPK/eh-fm/pull/44#discussion_r973617500)